v2/types.go

```diff
@@ -86,6 +86,28 @@ type Plan struct {
 	AlphaParameterSchemas *AlphaParameterSchemas `json:"schemas,omitempty"`
 }
 
+// AlphaBinding is an existing binding to a service instance.
+type AlphaBinding struct {
```

Comment:
I think this should be called `GetBindingResponse`, no `Alpha` prefix.

Reply:
And there should be a note that this is a distinct type because it contains fields both on a typical Binding and Parameters.  Also, you might be able to just embed the `BindingResponse` type.

Reply:
Sounds good to me.